public/department/sidebar.yaml

```
@@ -120,10 +120,26 @@ sections:
             label_ar: دفعة ٢١
             icon: MdAdjust
             link: "Extras/GPs/G2021.yaml"
-      - label: Summer Courses
-        label_ar: تدريبات صيفية
-        icon: MdGolfCourse
-        link: "Extras/SummerCourses.yaml"
+      - label: Opportunities
+        label_ar: فرص مختلفة
+        icon: MdTrackChanges
+        items:
+          - label: Google Summer of Code
+            label_ar: صيف جوجل البرمجي 
+            icon: MdBeachAccess
+            link: "Extras/Opportunities/GSoC.md"
+          - label: Teaching Positions
+            label_ar: فرص كمدرس مساعد
```

I feel like "مدرس مساعد" restricts it on TAs only when it focuses on encouraging teaching in general and most of the points mentioned can be applied on both TAs and professors. 
```suggestion
            label_ar: فرص تدريس
```

---

Agree.